pkg/controllers/dynakube/activegate/internal/statefulset/reconciler.go

```diff
@@ -249,14 +249,14 @@ func (r *Reconciler) getAuthTokenValue() (string, error) {
 
 func (r *Reconciler) getDataFromCustomProperty(customProperties *dynatracev1beta1.DynaKubeValueSource) (string, error) {
 	if customProperties.ValueFrom != "" {
-		return secret.GetDataFromSecretName(r.apiReader, types.NamespacedName{Namespace: r.dynakube.Namespace, Name: customProperties.ValueFrom}, customproperties.DataKey, log)
+		return secret.GetDataFromSecretName(r.apiReader, types.NamespacedName{Namespace: r.dynakube.Namespace, Name: customProperties.ValueFrom}, customproperties.DataKey, log.Logger)
```

Comment:
our code, we can expect our logger, it uses the `secret.NewQuery` fixing that will force you to fix this 😅

(https://github.com/Dynatrace/dynatrace-operator/blob/9a3bcd1bc6b3eb9f946d056c83474efb23dde74b/pkg/util/kubeobjects/secret/secret.go#L177-L177)